### PR TITLE
Docker implementation and Embedded Code.

### PR DESCRIPTION
**Pull request by Cruiz102**
Implementing #7

### CODE REVIEW

**Comment by Cruiz102** [comment]
The CI/DI pipeline is still not implemented, and the Jetson's Docker Compose hasn’t been completely tested yet. Additionally, there isn’t a main executable to test yet. It's still a work in progress, and I will keep everyone updated here.

**Comment by Cruiz102** [comment]
#7 is probably going to take a little bit more of time and we need this Pull Request for developers to use the dockers images. Please check the code and Merge it to main when possible. @ultra-azu @diegoq48  @JuanDelPueblo

File: docker/amd64/camera.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+FROM stereolabs/zed:4.1-devel-cuda12.1-ubuntu20.04
+
+# Print Ubuntu version
+RUN apt-get update && apt-get install -y lsb-release gnupg
+
+#Install ROS
+RUN sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+RUN curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
+RUN apt-get update && apt-get install -y ros-noetic-ros-base
```
Comment (JuanDelPueblo) [inline docker/amd64/camera.Dockerfile:9]: could merge these two apt-get run lines together

**Review by diegoq48 — APPROVED** [review]
Looks good would add less generic comments

**Pull request merged**